cmd/update/volume_move.go

```diff
@@ -43,6 +43,10 @@ type volumeMoveCommand struct {
 }
 
 func (c *volumeMoveCommand) runWithCtx(ctx context.Context, cmd *cobra.Command, args []string) error {
+	// this operation can run for a long time due to a sync operation
+	ctxLongTimeout, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
```

Comment:
We might want to make this configurable. 10 minutes might not be enough for some large volumes. It's probably worth adding a flag to control this

Reply:
Had a second look and timeout is a global flag so I will just remove this